Monika After Story/game/event-handler.rpy

```diff
@@ -513,6 +513,28 @@ init python:
         return cleanlist
 
 
+    def mas_cleanJustSeenEV(ev_list):
+        """
+        Cleans the given event list (of events) of just seen items
+        (within the THRESHOLD). Returns not just seen items.
+        Basically the same as mas_cleanJustSeen, except for Event object lists
+
+        IN:
+            ev_list - list of event objects
+
+        RETURNS:
+            cleaned list of events (stuff not in the tiem THRESHOLD)
+        """
+        import datetime
+        now = datetime.datetime.now()
+        
+        return [
+            ev
+            for ev in ev_list
+            if now - ev.last_seen >= store.evhand.LAST_SEEN_DELTA
```

Comment:
This one throws an error if `ev.last_seen` is None so it should be:
`if ev.last_seen is not None and now - ev.last_seen >= store.evhand.LAST_SEEN_DELTA`